src/executor/contract.rs

```diff
@@ -276,15 +276,15 @@ impl AotContractExecutor {
     /// - selector: The selector of the entry point to run.
     /// - args: The calldata.
     /// - gas: The gas for the execution.
-    /// - initial_gas_cost: is the gas cost for the called entry point, usually 10k ( `INITIAL_GAS_COST` ).
     /// - builtin_costs: An optional argument to customize the costs of the builtins.
     /// - syscall_handler: The syscall handler implementation to use when executing the contract.
+    ///
+    /// This method doesn't do any gas accounting, meaning it wont substract the initial gas cost of the entry point from the given gas.
```

Comment:
"This method doesn't do any gas accounting, meaning it wont substract the initial gas cost of the entry point from the given gas."

This comment is misleading since there is gas accounting during the execution.
What about:

"This entry point gas cost is not deducted from the gas counter."

Reply:
> "This method doesn't do any gas accounting, meaning it wont substract the initial gas cost of the entry point from the given gas."
> 
> This comment is misleading since there is gas accounting during the execution. What about:
> 
> "This entry point gas cost is not deducted from the gas counter."

changed!